src/features/network-modifications/twoWindingsTransformer/tapChanger/RegulatedTerminalSection.tsx

```
@@ -0,0 +1,90 @@
+    previousValues,
+    tapChangerDisabled,
+    regulationType,
+    fetchVoltageLevelEquipments,
+}: Readonly<RegulatedTerminalSectionProps>) {
+    const intl = useIntl();
+    const tapChangerPreviousValues =
+        id === FieldConstants.PHASE_TAP_CHANGER
+            ? previousValues?.ratioTapChanger
+            : id === FieldConstants.PHASE_TAP_CHANGER
+              ? previousValues?.phaseTapChanger
+              : undefined;
```

ESLint

[error] Do not nest ternary expressions.

(no-nested-ternary)